models/organization.js

```diff
@@ -16,15 +16,35 @@ module.exports = function(db) {
     description: {
       type: Sequelize.STRING(256),
       allowNull: true
-    }
+    },
+    twitter: {
+      type: Sequelize.STRING(256),
+      allowNull: true
+    },
+    facebook: {
+      type: Sequelize.STRING(256),
+      allowNull: true
+    },
+    instagram: {
+      type: Sequelize.STRING(256),
+      allowNull: true
+    },
+    createdAt: Sequelize.DATE,
```

Comment:
You don't need to expose the createdAt and updatedAt to the model. Since we'll never use this data in the front end, this just becomes useless data taking bytes of transfer.